etc/deploy/deploy.mizar.yaml

```diff
@@ -0,0 +1,492 @@
+---
```

Comment:
Does this replace deploy.mizar.components.yaml file?

Reply:
Yes it does, thanks for the catch
I will remove the old file and version the new docker images accordingly